src/providers/AccessControlProviderCustom.ts

```
@@ -44,6 +45,17 @@ export const AccessControlProviderCustom: AccessControlProvider = {
             const targetUserRole = params?.data?.role as UserRole;
             const targetUserMajor = params?.data?.major;
 
+            // TM và GV chỉ có thể edit bản thân (không bao gồm role, username, status)
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)